FreeRTOS/FreeRTOS-Plus-TCP
# Mirsa Rule 10.3 violations

Mirsa Rule 10.3 violations

Description
-----------
fix Mirsa Rule 10.3 violations

Test Steps
-----------
Violations disappeared from the report


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
